extensions/network/net-chrony.sh

```diff
@@ -0,0 +1,7 @@
+#
+# Extension to manage network time synchronization with Chrony
+#
+function add_host_dependencies__install_chrony() {
```

Comment:
`add_host_dependencies` is not adequate for this. use `extension_prepare_config` or such.